### DISCUSSION TITLE
correct groups to contact at insurers to alert about problems

### DISCUSSION THREAD

**Discussion by nmayle** [CMSgov/price-transparency-guide#533] [General]
For those of you producing files for your plans, what would be the correct contact person (name of the role) or group to ask for if we need to we reach out to your company to alert you to problems in an MRF? 

We would much rather report an issue to the people who can fix the problem than file a complaint which I assume will be a slow process. These files and format is all new and there are likely to be bugs even with careful QA processes.

Some pages that host files list a contact person or email address, but most do not. It is too bad the rule didn't require including a the email or phone number for a contact person.

**Comment by shaselton-usds** [comment]
I would quickly suggest in addition, for those that are producing the files, to test out the [validator](https://github.com/CMSgov/price-transparency-guide-validator) for the various files (for the reason stated above) before posting any information here.

**Reply by nmayle** [reply]
Running the validator would be great. I assume that folks that are posting files have run their files through a validator during whatever QA processes they run. As a consumer of files, we can also run the validator, but without knowing who and how to contact people at the producer of the file, we would not be able to do much with the errors.

In addition to the kinds of syntactic issues that a validator can identify, there are semantic issues in some files. I was referring to these types of issues in my question.

Generally speaking, since files can have various sorts of errors that one hopes the producers would want to fix, it would be great to know how in general to reach those responsible for the files with what are typically very large organizations.

**Comment by RonPeters** [comment]
I agree. There doesn't seem to be any way to tell the entities that generate these files that they contain errors that can still pass cursory validation. There's no email address or feedback form. I think that also needs to be mandated as part of this standard.

For instance:
1. Cigna - The TOC file for 7/2022 referenced at https://www.cigna.com/legal/compliance/machine-readable-files has 75 URLs within that cannot be downloaded and return 403 Forbidden errors
2. Anthem - The TOC file for 7/2022 referenced at https://www.anthem.com/machine-readable-file/search/ has 558 URLs that return 403 Forbidden errors (turns out Anthem used the wrong domain for those links)
3. I'm seeing URLs that expire days before the end of the month, and weeks before the expiration of the TOC file. At the very least, the expiration should extend through the end of the month. 

If we can't download the files because of faults in the data, then that's a violation of the mandate. How do we close the loop on this?
